Examples/BasiciOS/main.m

```diff
@@ -0,0 +1,3 @@
+int main(int argc, char * argv[]) {
+    return 0;
```

Comment:
reference symbols from the SLKTextViewController as sanity check?

Reply:
@mikezucc good suggestion 👍